lib/src/main/java/io/ably/lib/types/ChannelOptions.java

```diff
@@ -22,9 +22,15 @@
 	public boolean encrypted;
 
 	public ChannelCipher getCipher() throws AblyException {
-		if(!encrypted) return null;
-		if(cipher != null) return cipher;
-		return (cipher = Crypto.getCipher(this));
+		if(!this.encrypted) {
+			return null;
```

Comment:
I agree with the change to add braces, but why the insistence on using `this` ?

Reply:
@paddybyers I use ``this.`` all the time when dealing with class variables. It just seems cleaner to me and its clear from the start that it is a class attribute. I don't like relying on colours :) 

I also do it automatically... in previous projects, the build would fail if ``this,`` was not used.

Reply:
Personally I don't like that. I prefer using `this` where necessary for disambiguation, but not unconditionally.